src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costsPerDay = 40;
+  const discount = 20;
+  const longDiscount = 50;
+
+  if (days >= 7) {
+    return days * costsPerDay - longDiscount;
+  } else if (days >= 3) {
+    return days * costsPerDay - discount;
+  }
+
+  return days * costsPerDay;
```

Comment:
It would be great to create a variable for `days * costsPerDay` as it duplicates three times